development-process.md

```diff
@@ -0,0 +1,144 @@
+step-by-step guide for new developers.
+
+# Tools
+
+- git
+- Zenhub Chrome extension (https://www.zenhub.io/)
+  - This extension adds the **Boards** and **Burndown** tabs to Github repos. We use it in the way
+  that you might use Jira or Trello. The **Boards** tab is particularly important as it adds a
+  'swim lane'-type view for open issues.
+
+# Coding Workflow
+
+
+### Step 1) Hacking
+
+#### &#10146; Clone the given repository from Github
+
+Example: `git clone https://github.com/mitodl/micromasters.git`
+
+#### &#10146; Create a branch for the given issue
+
+Assuming you're working on issue **#123** in Github, you create a new branch from master like so:
+
+    git checkout -b NEW-BRANCH_NAME origin/master
+
+Eventually you'll want to push this new branch to origin to make it available to other developers.
+You can do this before you open a Pull Request (see below) or right after you create the branch:
+
+    git push -u origin NEW-BRANCH_NAME
+
+#### &#10146; Hack away!
+
+Make changes, test (automated and/or manual), commit, repeat. The repo README should have details
+for running automated tests.
+
+
+### Step 2) Code Review
+
+_When you've completed the given issue, the code is passing tests, and you feel like it's ready
+to ship, it's time to start code review. Make sure you read through the
+[Code Review guide](https://github.com/mitodl/handbook/blob/master/code_review.md) to get a sense
+about why we do code reviews and how to approach them, both as a reviewer and reviewee._
+
+#### &#10146; Open a Pull Request ("PR")
+
+This can be kicked off from several places in Github. The easiest one to remember is the "New
+Pull Request" button in your repo's "Pull Requests" tab
+([example](https://github.com/mitodl/micromasters/pulls)). Your **base** branch should be
+**master** and your **compare** branch should be your issue branch.
+
+When you first open the PR, the input should be pre-populated with our
+[PR template](https://github.com/mitodl/handbook/blob/master/pr-template.md). Fill in every section
+that is relevant to your PR. Under the **"relevant tickets"** section, be sure to indicate the
+issue that you're fixing. Github will automatically link to your issue if you type '#' followed
+by your issue number (and it will help you with an autocomplete menu). **If your PR is intended to
+fix/close an issue (typically true), fill in this section with "fixes #123" or "closes #123".**
+When your PR is eventually merged, this text is parsed, and the issue will be automatically closed
+if "fixes #X"/"closes #X" is found.
+
+Before (or after) you submit the PR, add the "Needs Review" label to it.
+
+**TL;DR**
+1. Click "New Pull Request"
+1. Fill in (or confirm) the appropriate **base** and **compare** branches
+1. Fill in all the relevant sections in the PR template (especially "relevant tickets", which should
+   typically read "fixes #<issue_number>")
+1. Add the "Needs Review" label
+1. Submit and wait for comments/approval
```

Comment:
I would put the TL;DR before the other, otherwise I do not think it is much useful (or just do not call it TL;DR)